app.py

```diff
@@ -46,6 +47,42 @@ def scrape_airbnb_reviews(url):
     finally:
         driver.quit()
 
+def scrape_airbnb_amenities(url):
+    driver = connect_browser()
+    
+    try:
+        driver.get(url)
+        WebDriverWait(driver, 60).until(EC.presence_of_element_located((By.XPATH, "//button[contains(text(), 'Show all 41 amenities')]")))
```

Comment:
this code will fail when a property doesn't have 41 amenities. need ot make it generic

Reply:
Fixed the code